pkg/config/types.go

```diff
@@ -31,6 +31,7 @@ const (
 	CPURefCycle    = "cpu_ref_cycles"
 	CPUInstruction = "cpu_instructions"
 	CacheMiss      = "cache_miss"
+	TaskClock      = "task_clock"
 
 	// bpf - attacher package
 	CPUTime       = "bpf_cpu_time_us"
```

Comment:
Shouldn't the CPUTime be replaced with TaskClock?

Reply:
added new metrics